examples/model_selection/plot_precision_recall.py

```diff
@@ -139,20 +139,35 @@
 plt.legend(loc="lower left")
 plt.show()
 
-# Plot Precision-Recall curve for each class
+# Plot Precision-Recall curve for each class and iso-f1 curves
 plt.clf()
+f_scores = np.linspace(0.2, 0.8, num=4)
+lines = []
+labels = []
+for f_score in f_scores:
+    x = np.linspace(0.01, 1)
+    y = f_score * x / (2 * x - f_score)
+    l, = plt.plot(x[y >= 0], y[y >= 0], color='gray', alpha=0.2)
```

Comment:
without the `,` , l becomes `[<matplotlib.lines.Line2D object at 0x7f43d13912e8>]` , which is not supported by `legend` (a UserWarning is raised). Using a `,` returns a line instance `Line2D(_line3)` which is what the `legend` function accepts! So, the lines have to be appended separately.